src/main/java/org/jitsi/jicofo/Participant.java

```diff
@@ -112,7 +113,7 @@
      * returns <tt>false</tt>.
      */
     private Map<String, IceUdpTransportPacketExtension> transportMap
-        = new HashMap<>();
+        = new ConcurrentHashMap<>();
```

Comment:
My idea was to protect the individual calls to get() and put(), since the map is exposed through getTransportMap().